---
layout: research
title: Research
permalink: /research/
---

## Sequencing and characterization of the gopher tortoise genome
![Gopher Tortoise - source: commons.wikimedia.org](https://upload.wikimedia.org/wikipedia/commons/thumb/1/10/Florida_Gopher_Tortoise.jpg/800px-Florida_Gopher_Tortoise.jpg)  
The gopher tortoise (*Gopherus polyphemus*) is native to the southeastern United States and is considered a keystone species for their roles in ecosystems such as pine savannas, which are highly abundant in Florida. The gopher tortoise is named for it's burrowing behavior. It digs burrows for shelter, where it is protected from the predators and extreme temperatures. These burrows are used by a number of other species for shelter. The gopher tortoise is classified as a "threatened" species by the U.S. Fish and Wildlife Service because of habitat loss, death from automobiles, and predation by humans. The FSU Center for Genomics and Personalized Medicine launched a crowdsourcing campaign through [sparkFSU](https://spark.fsu.edu/) to raise funds to sequence and characterize the genomes of the gopher tortoise and other threatened or endangered Florida species. The sequencing and characterization of genomes for the purpose of assisting conservation efforts, also known as "conservation genomics", provides molecular signatures that can be utilized to study and monitor gopher tortoise populations which are vital to help define appropriate actions for species conservation efforts. In addition, the genome can provide insights into the unique biological characteristics of the gopher tortoise and other endangered or threatened species. This project is being performed by undergraduate and graduate students at FSU under the supervision of the Director. Through this project, students are learning to perform whole-genome sequencing, from the extraction of DNA to the characterization and annotation of genomes. We would like to thank those who donated to this initiative which would not have been possible otherwise.

<div class="hline"></div>

## Origin and significance of altered replication timing in pediatric leukemia

B-lineage acute lymphocytic leukemia (B-ALL) is the most common childhood malignancy, yet we still do not understand the molecular mechanisms that lead to the genesis of this disease and there are no known strategies for its prevention. Pediatric B-ALL cells deviate from non-leukemic human B cells in the temporal order in which segments of their chromosomal DNA are duplicated (replication timing. The laboratory of FSU Professor of Biological Science Dave Gilbert, in collaboration with the Children’s Oncology Group and the FSU CGPM, is generating genome-wide replication timing profiles from a large cohort of pediatric B-ALL patients lacking prognostic markers and to potentially identify a new biomarkers for pediatric leukemia. In addition, we are profiling the changes in replication timing that accompany the process by which normal human B-cells are generated to determine which replication tming alterations in leukemia derive from the cell type of origin. Finally, we are developing a novel model of leukemic transformation of normal human hematopoietic cells to assess when and in what cell types evidence of “aberrant replication timing" first appears during the process of leukemogenesis. Replication timing has the potential to provide a new genre of biomarkers for diagnosis. In addition, replication timing reports on an undeveloped aspect of chromosome biology that is altered in B-ALL. Hence, studies of the cellular origin and mechanistic determinants of these replication timing alterations will provide novel insights into the underlying mechanisms of B-ALL.

<div class="hline"></div>

## Nuclease Profiling as an Integrative Resource for Maize Epigenomics
![Field of corn in Tallahassee, Florida](http://bio.fsu.edu/bass/img/bannerhome.jpg)

The eukaryotic nuclear genome is organized into nucleosomes which are segments of DNA that are wrapped around sets of proteins called histone octamers. Nucleosomes regulate the access of proteins to DNA, and thus can influence DNA-templated events such as transcription, DNA repair, DNA replication, and recombination. Nucleosomes regulate and respond to various nuclear processes through their position, modifications, and remodeling. FSU Professor of Biological Science Hank Bass, in collaboration with FSU CGPM Director Daniel Vera, are identifying nucleosome positions genomewide in multiple maize tissues to develop a foundational resource for maize epigenomic research. This collaborative project is also led by FSU Associate Professor of Biological Science Jonathan Dennis, FSU Associate Professor of Statistics Jinfeng Zhang, FAMU Professor of Viticulture and Developmental Biology Violeta Tsolova, and FAMU Professor of Agronomy Oghenekome Onokpise. This project builds on methods developed by Bass and Vera from a previous NSF plant genome project. Summer workshops will provide other researchers with new genome research expertise for their own research programs.

<div class="hline"></div>

## Chromatin structural changes linking drugs of abuse with HIV reactivation

Drug abuse and addiction add a significant challenging dimension to our comprehension of the progression and treatment of HIV/AIDS. A promising class of HIV treatments (Histone Deacetylase Inhibitors, HDIs) and psychostimulant drugs of abuse share a common denominator: both cause changes in chromatin structure through the acetylation of histones. Our goal is to identify and analyze the functional chromatin regulatory networks associated with HDIs and psychostimulant treatment by Identifying HDI- and psychostimulant-induced chromatin and gene regulatory changes, and define the role of HDI- and psychostimulant-induced chromatin structural changes in complex genetic and epigenetic regulatory networks.
